package.json

```diff
@@ -22,7 +21,7 @@
     "redux": "^4.0.5"
   },
   "scripts": {
-    "dev": "json-server --watch db.json -p 3001 & react-scripts start",
+    "dev": "json-server --watch db.json -p 3001 && start react-scripts start",
```

Comment:
is this right? you need start?

Reply:
yeah  we need start to start the second command